shardingsphere-ui/shardingsphere-ui-backend/src/main/java/org/apache/shardingsphere/ui/servcie/ShardingSchemaService.java

```diff
@@ -71,4 +71,11 @@
      * @param dataSourceConfiguration data source configuration
      */
     void addSchemaConfiguration(String schemaName, String ruleConfiguration, String dataSourceConfiguration);
+    
+    /**
+     * Delete schema configuration.
+     *
+     * @param schemaName schema name
+     */
+    void deleteSchemaConfiguration(String schemaName);
```

Comment:
deleteSchema is ok, 
AddSchemaConfiguration means add schema from configuration parameter.